test/jl/tests.jl

```diff
@@ -142,7 +142,7 @@ function test6()
    println("test 6: Conversion of images from Images.jl to OpenCV Mat")
    filename = joinpath(Pkg.dir("OpenCV"), "./test/images/mandrill.jpg")
    image = imread(filename)
-   dst = Mat(int(rows(image)), int(cols(image)), CV_8UC1)
+   dst = Mat(Int(round(rows(image))), Int(round(cols(image))), CV_8UC1)
    dst = convertToMat(image)
 end
 
```

Comment:
Test 5 : 
Video is streamed but I get this error in the terminal 
```julia
test 5: Basic video display: set video properties
Frames per second: 10.0
:1:1: error: no matching constructor for initialization of 'cv::Mat'
^
/usr/local/include/opencv2/core/mat.inl.hpp:339:6: note: candidate constructor not viable: no known conversion from 'long *' to 'int' for 2nd argument; dereference the argument with *
Mat::Mat(int _rows, int _cols, int _type)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:373:6: note: candidate constructor not viable: no known conversion from 'long *' to 'const int *' for 2nd argument
Mat::Mat(int _dims, const int* _sz, int _type)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:364:6: note: candidate constructor not viable: no known conversion from 'long' to 'Size' (aka 'Size_<int>') for 1st argument
Mat::Mat(Size _sz, int _type, const Scalar& _s)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:398:6: note: candidate constructor not viable: no known conversion from 'long' to 'const std::vector<int>' for 1st argument
Mat::Mat(const std::vector<int>& _sz, int _type, const Scalar& _s)
     ^
/usr/local/include/opencv2/core/mat.hpp:894:5: note: candidate constructor not viable: no known conversion from 'long' to 'const std::vector<int>' for 1st argument
    Mat(const std::vector<int>& sizes, int type, void* data, const size_t* steps=0);
    ^
/usr/local/include/opencv2/core/mat.hpp:906:5: note: candidate constructor not viable: no known conversion from 'long' to 'const cv::Mat' for 1st argument
    Mat(const Mat& m, const Range& rowRange, const Range& colRange=Range::all());
    ^
/usr/local/include/opencv2/core/mat.inl.hpp:459:6: note: candidate constructor not viable: no known conversion from 'long' to 'Size' (aka 'Size_<int>') for 1st argument
Mat::Mat(Size _sz, int _type, void* _data, size_t _step)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:492:6: note: candidate constructor template not viable: requires at most 2 arguments, but 3 were provided
Mat::Mat(const std::vector<_Tp>& vec, bool copyData)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:509:6: note: candidate constructor template not viable: requires at most 2 arguments, but 3 were provided
Mat::Mat(const Vec<_Tp, n>& vec, bool copyData)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:525:6: note: candidate constructor template not viable: requires at most 2 arguments, but 3 were provided
Mat::Mat(const Matx<_Tp,m,n>& M, bool copyData)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:541:6: note: candidate constructor template not viable: requires at most 2 arguments, but 3 were provided
Mat::Mat(const Point_<_Tp>& pt, bool copyData)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:560:6: note: candidate constructor template not viable: requires at most 2 arguments, but 3 were provided
Mat::Mat(const Point3_<_Tp>& pt, bool copyData)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:580:6: note: candidate constructor template not viable: requires single argument 'commaInitializer', but 3 arguments were provided
Mat::Mat(const MatCommaInitializer_<_Tp>& commaInitializer)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:390:6: note: candidate constructor not viable: requires 2 arguments, but 3 were provided
Mat::Mat(const std::vector<int>& _sz, int _type)
     ^
/usr/local/include/opencv2/core/mat.hpp:936:5: note: candidate constructor not viable: requires 2 arguments, but 3 were provided
    Mat(const Mat& m, const std::vector<Range>& ranges);
    ^
/usr/local/include/opencv2/core/mat.hpp:926:5: note: candidate constructor not viable: requires 2 arguments, but 3 were provided
    Mat(const Mat& m, const Range* ranges);
    ^
/usr/local/include/opencv2/core/mat.hpp:916:5: note: candidate constructor not viable: requires 2 arguments, but 3 were provided
    Mat(const Mat& m, const Rect& roi);
    ^
/usr/local/include/opencv2/core/mat.inl.hpp:356:6: note: candidate constructor not viable: requires 2 arguments, but 3 were provided
Mat::Mat(Size _sz, int _type)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:381:6: note: candidate constructor not viable: requires 4 arguments, but 3 were provided
Mat::Mat(int _dims, const int* _sz, int _type, const Scalar& _s)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:347:6: note: candidate constructor not viable: requires 4 arguments, but 3 were provided
Mat::Mat(int _rows, int _cols, int _type, const Scalar& _s)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:407:6: note: candidate constructor not viable: requires single argument 'm', but 3 arguments were provided
Mat::Mat(const Mat& m)
     ^
/usr/local/include/opencv2/core/cuda.inl.hpp:621:6: note: candidate constructor not viable: requires single argument 'm', but 3 arguments were provided
Mat::Mat(const cuda::GpuMat& m)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:1179:6: note: candidate constructor not viable: requires single argument 'm', but 3 arguments were provided
Mat::Mat(Mat&& m)
     ^
/usr/local/include/opencv2/core/mat.hpp:880:5: note: candidate constructor not viable: requires at least 4 arguments, but 3 were provided
    Mat(int ndims, const int* sizes, int type, void* data, const size_t* steps=0);
    ^
/usr/local/include/opencv2/core/mat.inl.hpp:426:6: note: candidate constructor not viable: requires at least 4 arguments, but 3 were provided
Mat::Mat(int _rows, int _cols, int _type, void* _data, size_t _step)
     ^
/usr/local/include/opencv2/core/mat.inl.hpp:333:6: note: candidate constructor not viable: requires 0 arguments, but 3 were provided
Mat::Mat()

```
and I was unable to test `Test 6`, there seems to be a bug with Images.jl which is used in convertToMat function for test 6. I have filed an issue [here](https://github.com/JuliaImages/Images.jl/issues/604).